Assets/HoloToolkit/Utilities/Editor/AutoConfigureMenu.cs

```diff
@@ -0,0 +1,182 @@
+        /// </summary>
+        [MenuItem("HoloToolkit/Auto Configure/Project", false)]
+        public static void ApplyProjectSettings()
+        {
+            // Build settings
+            EditorUserBuildSettings.SwitchActiveBuildTarget(BuildTarget.WSAPlayer);
+            EditorUserBuildSettings.wsaSDK = WSASDK.UWP;
+            EditorUserBuildSettings.wsaUWPBuildType = WSAUWPBuildType.D3D;
+
+            // See the blow notes for why text asset serialization is required
+            if (EditorSettings.serializationMode != SerializationMode.ForceText)
+            {
+                // NOTE: PlayerSettings.virtualRealitySupported would be ideal, except that it only reports/affects whatever platform tab
+                // is currently selected in the Player settings window. As we don't have code control over what view is selected there
+                // this property is fairly useless from script.
+
+                // NOTE: There is no current way to change the default quality setting from script
+
+                string title = "Updates require text serialization of assets";
+                string message = "Unity doesn't provide apis for updating the default quality or enabling VR support.\n\n" +
+                    "Is it ok if we force text serialization of assets so that we can modify the properties directly?";
+
+                bool forceText = EditorUtility.DisplayDialog(title, message, "Yes", "No");
+                if (!forceText)
+                    return;
+
+                EditorSettings.serializationMode = SerializationMode.ForceText;
+            }
+
+            SetFastestDefaultQuality();
+            EnableVirtualReality();
+
+            // Since we went behind Unity's back to tweak some settings we 
+            // need to reload the project to have them take effect
+            bool canReload = EditorUtility.DisplayDialog(
+                "Project reload required!",
+                "Some changes require a project reload to take effect.\n\nReload now?",
+                "Yes", "No");
+
+            if (canReload)
+                EditorApplication.OpenProject(Application.dataPath + "/..");
+        }
+
+        /// <summary>
+        /// Modifies the WSA default quality setting to the fastest
+        /// </summary>
+        private static void SetFastestDefaultQuality()
+        {
+            try
+            {
+                // Find the WSA element under the platform quality list and replace it's value with 0
+                string settingsPath = "ProjectSettings/QualitySettings.asset";
+                string matchPattern = @"(m_PerPlatformDefaultQuality.*Windows Store Apps:) (\d+)";
+                string replacePattern = @"$1 0";
+
+                string settings = File.ReadAllText(settingsPath);
+                settings = Regex.Replace(settings, matchPattern, replacePattern, RegexOptions.Singleline);
+
+                //Debug.Log("Updated Quality Settings: " + settings);
```

Comment:
Nitpick: Delete this. Or make a nicely-formatted log entry the describes all the changes that were made.